src/components/shared/Carousel.tsx

```diff
@@ -22,6 +22,7 @@ export interface CarouselItemComponents {
 }
 
 interface CarouselProps {
+  redirect: any;
```

Comment:
```suggestion
  redirect: () => void;
```

or better yet, find the React.Dispatch type for this

Reply:
Ok, I set it to void right now but ill look into the Dispatch type!